Specs/Scene/ExpressionSpec.js

```diff
@@ -51,6 +51,12 @@ defineSuite([
 
         expression = new Expression(new MockStyleEngine(), 'hsl(0, 0, 1)');
         expect(expression.evaluate(undefined)).toEqual(Color.WHITE);
+
+        expression = new Expression(new MockStyleEngine(), 'rgba(0, 0, 0, 0)');
```

Comment:
These alpha tests are a bit too trivial.  The `expect` doesn't need to be a `Color` constant, it can be a `Color` you construct (might need to use `Color.equals`).